prow/pod-utils/clone/clone.go

```diff
@@ -57,13 +57,22 @@ func Run(refs prowapi.Refs, dir, gitUserName, gitUserEmail, cookiePath string, e
 	if err := runCommands(g.commandsForBaseRef(refs, gitUserName, gitUserEmail, cookiePath)); err != nil {
 		return record
 	}
+
 	timestamp, err := g.gitHeadTimestamp()
 	if err != nil {
 		timestamp = int(time.Now().Unix())
 	}
 	if err := runCommands(g.commandsForPullRefs(refs, timestamp)); err != nil {
 		return record
 	}
+
+	finalSHA, err := g.gitRevParse()
+	if err != nil {
+		logrus.Warnf("Cannot resolve baseSHA for ref %#v", refs)
```

Comment:
This log message doesn't seem accurate.
Please also log the `err` itself.

Reply:
🤦‍♂️ bad copy pasta, done